packages/from-query/src/generate.ts

```diff
@@ -66,7 +66,20 @@ export default (OPTIONS: IFromQueryOptions): (ir: IOperation) => string => {
     }
   };
 
-  const printField: (name: string, type: TypeDefinition | string, node: Selection, nameOverride?: string) => string = (name, type, node, nameOverride) => `${name}: ${printType(type, node, nameOverride)};`;
+  const hasDirectives: (directives: IDirectiveMap) => boolean = directives =>
+    !!directives['include'] || !!directives['skip']
```

Comment:
```suggestion
    !!directives.include || !!directives.skip
```

you can use dot-notation